code/modules/antagonists/clock_cult/items/clockwork_slab.dm

```diff
@@ -210,3 +223,28 @@ GLOBAL_LIST_INIT(clockwork_slabs, list())
 			//Create and assign the quickbind
 			var/datum/clockcult/scripture/new_scripture = new S.type()
 			bind_spell(M, new_scripture, positions.Find(position))
+
+// Golem internal clockwork slab
+
+/obj/item/clockwork/clockwork_slab/empowered/ui_state(mob/user)
+	return GLOB.conscious_state
+
```

Comment:
Check returned: Line 52: Warning: Proc with an important parent call (Line 46) (new/destroy/init) does not call ..()! This will cause the object to fail to initialize or delete which is bad.